windows/src/engine/mcompile/mcompile.cpp

```diff
@@ -474,3 +487,8 @@ void LogError(PWSTR fmt, ...) {
 	fmtbuf[255] = 0;
   _putws(fmtbuf);
 }
+
+BOOL CopyKeyboard(PWSTR source, PWSTR target)
+{
+  return CopyFileW(source, target, FALSE /*overwrite*/);
```

Comment:
```suggestion
  return CopyFile(source, target, TRUE /*overwrite*/);
```

I think we should use the same overwrite semantics that `SaveKeyboard` uses (it calls [`CreateFile`](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea) with `CREATE_ALWAYS`).

We don't need the `W` suffix because we are compiling with `#define UNICODE`, and we are not using it in most other places.

Reply:
- I saw a warning without the W, but could have been intellisense getting confused.
- Is the target file expected to not already exist? I wasn't sure.

Reply:
Target file could exist for a reinstall or where there is install over broken earlier install (which has been known to happen). We are happy to clobber existing file in either situation.

Reply:
yeah, the 3rd argument is `bFailIfExists` which needs to be `FALSE` for overwriting. Updated the comment, because I see how my previous comment might look like the _argument_ was 'overwrite' when it's the opposite.

Reply:
Ah ... yes, sorry my bad!